### PR TITLE
[T4A5][F09-B2]AntoniaDevina

### PR DESCRIPTION
**Pull request by antoniadevina**

### CODE REVIEW

**Comment by antoniadevina** [comment]
ready for review

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+public class Tagging {
+	
+	private static StringBuilder fullTaggingList = new StringBuilder ();
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java:5]: Putting the stringBuilder here doesn't follow the diagram, you should declare it as an object in `AddressBook` class, check #1618.

File: src/seedu/addressbook/Main.java [hunk 2]
```diff
@@ -98,6 +99,7 @@ private void runCommandLoopUntilExitCommand() {
 			ui.showResultToUser(result);
 
 		} while (!ExitCommand.isExit(command));
+		System.out.println(Tagging.getFullTaggingList());
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/Main.java:102]: Here you doesn't follow SLAP.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		for (Tag tag : tags) { 
+			Tagging.modifyTaggingList (name, tag.toString() ,true);
+        }
+	}
+	
+	public static void modifyTaggingList (String name, String tag, boolean isAdded){ 
+		if (isAdded){ 
+			fullTaggingList.append("+ ");
+		}
+		else {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java:28]: Please follow our coding convention..

File: src/seedu/addressbook/data/tag/UniqueTagList.java [hunk 4]
```diff
@@ -67,9 +67,10 @@ public UniqueTagList(Set<Tag> tags) {
      * Constructs a shallow copy of the given tag list.
      */
     public UniqueTagList(UniqueTagList source) {
-        internalList.addAll(source.internalList);
-    }
+    	internalList.addAll(source.internalList);
 
+    }
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/UniqueTagList.java:73]: No changes here, shouldn't appear in this commit.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java:2]: You forgot to put the header comment.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+//	public Tagging (Person person, Tag tag, boolean isAdded){
+//		this.person = person; 
+//		this.tag = tag; 
+//		this.isAdded = isAdded;
+//	}
+	
+	public static void addAllTagsToTheEndMassage(String name, UniqueTagList tags) {
+		for (Tag tag : tags) { 
+			Tagging.modifyTaggingList (name, tag.toString() ,true);
+        }
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java:21]: Don't forget to fix the indentation.

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**